templates/clusterclass/clusterclass.yaml

```diff
@@ -37,377 +37,414 @@ spec:
           timeout: 300s
   workers:
     machineDeployments:
-    - class: nutanix-quick-start-worker
-      template:
-        bootstrap:
-          ref:
-            apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
-            kind: KubeadmConfigTemplate
-            name: "nutanix-quick-start-kcfg-0"
-        infrastructure:
-          ref:
-            apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-            kind: NutanixMachineTemplate
-            name: nutanix-quick-start-md-nmt
-      machineHealthCheck:
-        maxUnhealthy: 40%
-        nodeStartupTimeout: 10m
-        unhealthyConditions:
-          - type: Ready
-            status: "False"
-            timeout: 300s
-          - type: Ready
-            status: Unknown
-            timeout: 300s
-          - type: MemoryPressure
-            status: "True"
-            timeout: 300s
-          - type: DiskPressure
-            status: "True"
-            timeout: 300s
-          - type: PIDPressure
-            status: "True"
-            timeout: 300s
-          - type: NetworkUnavailable
-            status: "True"
-            timeout: 300s
+      - class: nutanix-quick-start-worker
```

Comment:
is this change required?

Reply:
Good catch, no it isn't. I think my IDE auto indented it.